docs/source/tutorial/gpu.rst

```diff
@@ -153,27 +153,6 @@ In this subsection, the code is based on :ref:`our first MNIST example in this t
 
 A :class:`Link` object can be transferred to the specified GPU using the :meth:`~Link.to_gpu` method.
 
-.. testcode::
```

Comment:
Do not remove this test code.
Please fix this.

You can use command to test for documents.
```bash
cd docs
make doctest
```